src/pages/guidelines/accessibility/developers.mdx

```diff
@@ -165,13 +175,13 @@ Add styling to tabbable elements on hover and focus, so that keyboard-only users
 
 When hiding content from a screen reader, consider source order. Use `visibility: hidden`, along with `display: none` in your CSS.
 
-## JS best practices
+## JavaScript best practices
 
-### JavaScript rule of thumb
+Don't rely too much on JavaScript to generate HTML and CSS. Follow the "Unobtrusive JavaScript" principle, using JavaScript to enhance functionality without relying on it to build your experience. Use your browser's built-in functionality as much as possible. If you're utilizing JavaScript to make complex UI features, use WAI-ARIA to make elements accessible.
```

Comment:
This might land a little awkwardly given our requirement on React to render 😅

Reply:
We can ditch this rule of thumb if you don't find it an accurate reflection of our dev approach